Makefile

```diff
@@ -1,9 +1,9 @@
 .PHONY: all test test-local install-deps lint fmt vet
 
 REPO_NAME = go-plugins-helpers
-REPO_OWNER = docker
+REPO_OWNER = Scalingo
```

Comment:
This change is mandatory for the tests to be executable lcoally with the Makefile. But the Makefile uses Docker which is not usable in CodeShip.